django/db/models/lookups.py

```diff
@@ -1,106 +1,27 @@
-import inspect
 from copy import copy
 
+from django.conf import settings
+from django.db.models.expressions import Func, Value
+from django.db.models.fields import (
+    DateField, DateTimeField, Field, IntegerField, TimeField,
+)
+# Maintain backwards compatibility by allowing RegisterLookupMixin
+# to be imported from here.
```

Comment:
use style for this type of comment is `# For backwards compatibility in Django 1.9`
(then we have some context in case we consider removing it in some future version)